cmd/server-main.go

```diff
@@ -363,8 +363,13 @@ func serverMain(c *cli.Context) {
 		cli.ShowCommandHelpAndExit(c, "server", 1)
 	}
 
-	// Set global quiet flag.
-	globalQuiet = c.Bool("quiet") || c.GlobalBool("quiet")
+	// Set global variables after parsing passed arguments
+	setGlobalsFromContext(c)
+
+	checkUpdate()
+
+	// Generic initialization
```

Comment:
Comment should talk about what it does.. "Generic initialization" is perhaps too "Generic"